main.go

```diff
@@ -119,7 +119,9 @@ func main() {
 	flag.StringVar(&opts.TLSKey, "tlskey", "", "Private key for server certificate.")
 	flag.StringVar(&opts.TLSCaCert, "tlscacert", "", "Client certificate CA for verification.")
 
-	flag.Usage = usage
+	flag.Usage = func() {
+		fmt.Printf("%s\n", usageStr)
```

Comment:
I did a `git push --force`. The usage function is still there (top of the file) and I have put back the `os.Exit(0)` call. Again, we call `usage()` directly if user provides `help` command line param.